salt/utils/reactor.py

```diff
@@ -417,7 +417,16 @@ def run(self, low):
             # and kwargs['kwarg'] contain the positional and keyword arguments
             # that will be passed to the client interface to execute the
             # desired runner/wheel/remote-exec/etc. function.
-            l_fun(*args, **kwargs)
+            ret = l_fun(*args, **kwargs)
+
+            if ret is False:
+                log.error('Reactor \'%s\' failed  to execute %s \'%s\': '
+                            'TaskPool queue is full!'
```

Comment:
There is no space between this sentence and the next. A space should be added before `Consider`.

Reply:
Done

Reply:
Sure, from what I can see, False is only returned when the queue is full. As per current behavior, we will right now silently fail to insert something on the queue. The only behavior I am adding is a log message when the queue is full. Otherwise, continue as per today.